Themes/default/css/index.css

```diff
@@ -135,6 +135,7 @@ input[type="checkbox"], input[type="radio"] {
 	border: none;
 	background: none;
 	vertical-align: middle;
+	margin: 0 5px 0 0;
```

Comment:
Can you move this somewhere more specific? It's going to add odd margins elsewhere (ie: post checkboxes)